packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -138,7 +142,9 @@ export class FocusZone extends React.Component<FocusZoneProps> implements IFocus
     as: 'div',
     preventDefaultWhenHandled: true,
     shouldRaiseClicks: false,
-  };
+    // Hardcoding uncontrolled flag for proper interop with FluentUI V9.
+    'data-tabster': '{"uncontrolled": {}}',
```

Comment:
Why it's not the same as for v0?

Reply:
What do you mean not the same? There are two FocusZones and two FocusTrapZones. FocusZones have `{"uncontrolled": {}}` and FocusTrapZones have `{"uncontrolled": {"completely": true}}`.

Reply:
Ah, I see. Sorry for confusiom